addons/mail/static/src/js/activity.js

```diff
@@ -740,9 +752,49 @@ var KanbanActivity = BasicActivity.extend({
     },
 });
 
+// -----------------------------------------------------------------------------
+// Activity Exception Widget to display Exception icon ('activity_exception' widget)
+// -----------------------------------------------------------------------------
+
+var ActivityException = AbstractField.extend({
```

Comment:
Can you add some QUnit tests for this new widget?

Reply:
@mge-odoo  Done.